### PR TITLE
[REF:CreateChallenge.vue|SingleChallenge.vue|theme.scss] added new fi…

### PR DESCRIPTION
**Pull request by aman2712**
…elds to forms

### CODE REVIEW

**Comment by ShantanuLamba** [comment]
Everything is fine !

File: frontend/src/pages/create-challenge/CreateChallenge.vue [hunk 1]
```diff
@@ -5,8 +5,25 @@
         <p class="text-muted text-center mb-3">Create a challenge for others to face!</p>
         <b-form class="px-0 px-md-5">
           <b-form-group id="input-group-1" class="px-1 px-md-5">
-            <b-form-textarea rows="10" name="text" placeholder="Your awesome challenge here" />
-            <b-button variant="primary" class="mt-2 w-100 mx-auto">Submit</b-button>
+            <!-- <div class="d-md-flex p-2"> -->
```
Comment (Wittmaxi) [inline frontend/src/pages/create-challenge/CreateChallenge.vue:8]: No commented-out code! 

https://kentcdodds.com/blog/please-dont-commit-commented-out-code

File: frontend/src/pages/single-challenge/SingleChallenge.vue [hunk 2]
```diff
@@ -9,29 +9,61 @@
...
+                {
+                    email: 'edf456@gmail.com',
+                    text: 'Lorem, ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quasi distinctio placeat, nemo fuga nobis deserunt voluptas sit nihil. Voluptatem.'
+                }
+            ]
+        }
+    },
+    methods: {
+        handleSubmit: function () {
+            console.log('email >>>', this.form.email)
```
Comment (Wittmaxi) [inline frontend/src/pages/single-challenge/SingleChallenge.vue:62]: no console log in commits (unless required by implementation)

File: frontend/src/pages/single-challenge/SingleChallenge.vue [hunk 3]
```diff
@@ -9,29 +9,61 @@
...
-          </div>
-          <div class="mt-3 my-2">
-              <p class="text-muted m-0">edf456@gmail.com</p>
-              <p class="text-white">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quasi distinctio placeat, nemo fuga nobis deserunt voluptas sit nihil. Voluptatem.</p>
-          </div>
-          <div class="mt-3 my-2">
-              <p class="text-muted m-0">edf456@gmail.com</p>
-              <p class="text-white">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Cupiditate quasi distinctio placeat, nemo fuga nobis deserunt voluptas sit nihil. Voluptatem.</p>
+          <div class="mb-5">
+              <div v-for="item in comments" :key="item.index" :class="{'mt-3': true, 'my-2': true, 'border-bottom': (comments.indexOf(item) + 1 !== comments.length), 'border-secondary': true}">
```
Comment (Wittmaxi) [inline frontend/src/pages/single-challenge/SingleChallenge.vue:25]: You can mix `class` and `:class`

Please use the former for styles that always apply!

File: frontend/src/pages/single-challenge/SingleChallenge.vue [hunk 4]
```diff
@@ -9,29 +9,61 @@
           <h3 class="mt-4 text-white">Your Answers</h3>
           <b-form class="d-flex flex-column">
               <b-form-group>
-                <b-form-textarea rows="3" name="text" placeholder="Leave your answer here.." />
+                <b-row>
```
Comment (Wittmaxi) [inline frontend/src/pages/single-challenge/SingleChallenge.vue:12]: This file should be in the folder for challenges, not in a separate folder!

**Review by Wittmaxi — CHANGES_REQUESTED** [review]

File: frontend/src/pages/challenges/create-challenge/CreateChallenge.vue [hunk 5]
```diff
@@ -0,0 +1,50 @@
+<template>
+  <b-container>
+      <div class="px-3 py-2">
+        <h1 class="text-center mt-2 text-white">Create a challenge</h1>
+        <p class="text-muted text-center mb-3">Create a challenge for others to face!</p>
+        <b-form class="px-0 px-md-5">
+          <b-form-group id="input-group-1" class="px-1 px-md-5">
+              <b-row>
+                <b-col lg='6' md='12'>
```
Comment (Wittmaxi) [inline frontend/src/pages/challenges/create-challenge/CreateChallenge.vue:9]: please be consistent with " and '

File: frontend/src/theme.scss [hunk 6]
```diff
@@ -83,8 +83,8 @@ a:hover {
 // Colors
 
 $theme-colors: (
-  primary: #dbb880,
-  secondary: #c07095,
+  primary: #c07095,
```
Comment (Wittmaxi) [inline frontend/src/theme.scss:86]: ?

**Review by Wittmaxi — CHANGES_REQUESTED** [review]

**Pull request merged**